CHANGELOG.md

```diff
@@ -1,17 +1,21 @@
 # `chainweb-node` Changelog
 
-## Unreleased
+## 1.8 (TODO)
```

Comment:
We should just set this to like "a day before the kill switch" (2020-04-29) or something

Reply:
I set it to the planned release date.